app/package.json

```diff
@@ -16,7 +16,7 @@
         "react-native-fast-image": "^5.0.3",
         "react-native-fetch-blob": "^0.10.8",
         "react-native-image-zoom-viewer": "^2.2.5",
-        "react-native-phone-input": "^0.2.1",
+        "react-native-phone-input": "lbryio/react-native-phone-input",
```

Comment:
Can you make an upstream request?
We have a lot of repos and it would be great to get maintained upstream fixes

Reply:
Upstream was last updated 6 months ago and there's been no response from the repo owner. I was thinking we could probably have a separate github org for forked repos.

Reply:
Alternatively, and so long as it's not against the license to do so, just check the code directly into our repo.